http/src/main/resources/reference.conf

```diff
@@ -18,7 +18,7 @@ dataproc {
     "https://www.googleapis.com/auth/cloud.useraccounts.readonly",
     "https://www.googleapis.com/auth/bigquery"
   ],
-  welderDockerImage = "us.gcr.io/broad-dsp-gcr-public/welder-server:6036f82"
+  welderDockerImage = "us.gcr.io/broad-dsp-gcr-public/welder-server:60e28bc"
```

Comment:
I think there's an automation `reference.conf` to update too?

Reply:
updated!